pages/index.vue

```diff
@@ -13,7 +13,7 @@
 
             <v-row no-gutters class="ctas mb-4 mt-10 d-flex flex-wrap">
               <v-col class="mr-6" cols="12" :sm="8" :md="3">
-                <Button primary hero nuxt to="/pilot" class="mt-4">Start a Pilot</Button>
+                <Button primary hero nuxt to="/pilot" class="mt-4" style="text-transform:none;">Start a Pilot</Button>
```

Comment:
Shouldn't need this change w/ the `Button` updates in https://github.com/covid19risk/website/pull/37/files

Reply:
Great!  I removed all inline `text-transformation` lines from `pilot.vue`.  

Not that re: Button I added in this PR the `font-weight: bold` which is what the [Figma design](https://www.figma.com/file/UwY5hq1o8Lw9aPkBIZu7lc/Covid-Watch---Website-v2.0?node-id=1011%3A623) shows.

I made a few minor additional updates that I think are now truly, truly final,  assuming of course the Button and the` pageSection` Class mixin are not altered.

Feel free to test responsiveness etc and merge if you think it's ok!